docs/user-guide/commands.md

```diff
@@ -630,16 +634,21 @@ Usage:  watson stop [OPTIONS]
 
 Stop monitoring time for the current project.
 
+If '--at' option is given, the provided stopping time is used. The
+specified time must be after the begin of the to be ended frame and must
+not be in the future.
+
 Example:
 
 
-    $ watson stop
-    Stopping project apollo11, started a minute ago. (id: e7ccd52)
+    $ watson stop --at 13:37
+    Stopping project apollo11, started a minute ago, at 30 minutes ago. (id: e9ccd52) # noqa: E501
```

Comment:
I think we can manually remove the `# noqa: E501` comment :wink:

Reply:
Sorry, I don't get that. If I remove it, flke8 complains.

Reply:
I think @jmaupetit meant to remove it in the `.md` document, which isn't concerned by flake8 :) .

Reply:
Indeed :wink: